src/components/gltf-model-plus.js

```diff
@@ -301,6 +402,7 @@ AFRAME.registerComponent("gltf-model-plus", {
       this.el.setObject3D("mesh", object3DToSet);
       this.el.emit("model-loaded", { format: "gltf", model: this.model });
     } catch (e) {
+      delete GLTFCache[src];
```

Comment:
Probably not part of this PR since we weren't doing it before, but we never actually clean up the GLTFCache in non-error cases. For images I do a reference counting cache, we should maybe do that here as well.